src/main/java/org/example/app/solrclient/TwitSolrClient.java

```diff
@@ -0,0 +1,36 @@
+package org.example.app.solrclient;
+
+import org.apache.solr.client.solrj.SolrQuery;
+import org.apache.solr.client.solrj.SolrServerException;
+import org.apache.solr.client.solrj.impl.HttpSolrClient;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocumentList;
+import org.apache.solr.common.SolrInputDocument;
+import org.example.app.entity.Twit;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TwitSolrClient {
+    private final static String URL_SOLR = "http://localhost:8983/solr/twits";
```

Comment:
Try to avoid URLs hardcoding. We can move it to config file (application.yml for instance). You can use @Value annotation to set the value for this variable or smth like that

Reply:
fixed